products/BellHybrid/apps/application-bell-meditation-timer/presenter/MeditationTimerPresenter.cpp

```diff
@@ -14,12 +16,22 @@ namespace
     constexpr auto spinnerStep  = 1U;
     constexpr auto emptyValue   = 0U;
     constexpr auto defaultValue = 15U;
+    constexpr units::SOC lowBatteryThreshold{10};
```

Comment:
Here you can use `=` to assign the value like above ;)

Reply:
I prefer `{}` wherever possible :)